x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState } from 'react';
+import { EuiSpacer, EuiFieldSearch, EuiButtonEmpty, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
+import useDebounce from 'react-use/lib/useDebounce';
+import * as TEST_SUBJECTS from './test_subjects';
+import * as TEXT from './translations';
+import { RulesBulkActionsMenu } from './rules_bulk_actions_menu';
+
+interface RulesTableToolbarProps {
+  search(value: string): void;
+  refresh(): void;
+  bulkEnable(): void;
+  bulkDisable(): void;
+  selectAll(): void;
+  clearSelection(): void;
+  totalRulesCount: number;
+  selectedRulesCount: number;
+  searchValue: string;
+  isSearching: boolean;
+}
+
+interface CounterProps {
+  count: number;
+}
+
+interface ButtonProps {
+  onClick(): void;
+}
+
+export const RulesTableHeader = ({
+  search,
+  refresh,
+  bulkEnable,
+  bulkDisable,
+  selectAll,
+  clearSelection,
+  totalRulesCount,
+  selectedRulesCount,
+  searchValue,
+  isSearching,
+}: RulesTableToolbarProps) => (
+  <div>
+    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" responsive={false} wrap>
+      <Counters total={totalRulesCount} selected={selectedRulesCount} />
+      <SelectAllToggle
+        isSelectAll={selectedRulesCount === totalRulesCount}
+        clear={clearSelection}
+        select={selectAll}
+      />
+      <BulkMenu
+        bulkEnable={bulkEnable}
+        bulkDisable={bulkDisable}
+        selectedRulesCount={selectedRulesCount}
+      />
+      <RefreshButton onClick={refresh} />
+      <SearchField isSearching={isSearching} searchValue={searchValue} search={search} />
+    </EuiFlexGroup>
+    <EuiSpacer />
```

Comment:
This spacer does not belong here, it belongs in the component that composes the table header and the table itself (`RulesContainer`)